sql/core/src/main/scala/org/apache/spark/sql/execution/QueryExecution.scala

```diff
@@ -127,10 +131,14 @@ class QueryExecution(
     ReuseExchange(sparkSession.sessionState.conf),
     ReuseSubquery(sparkSession.sessionState.conf))
 
-  def simpleString: String = withRedaction {
+  def simpleString[T <: QueryPlan[T]]: String = withRedaction {
```

Comment:
why do we need type parameter here?

Reply:
@cloud-fan not required... thanks .. will remove.